modules/test/playwright/tests/frontend-data-set-views-web/pages/DataSetFragmentPage.ts

```diff
@@ -77,25 +83,24 @@ export class FDSFragmentPage {
 			this.page.getByText('Place fragments or widgets here')
 		);
 
-		const fragmentSelectionArea = this.page.getByText(
-			'Select a data set view'
-		);
+		const fragmentSelectionArea = this.page.getByText('Select a data set');
 
 		await expect(fragmentSelectionArea).toBeVisible();
 
 		await fragmentSelectionArea.click();
 
-		await this.page
-			.getByRole('button', {name: 'Select Data Set View'})
-			.click();
+		const dataSetSelectionTrigger = this.page.getByTitle(
+			'Select Data Set',
+			{exact: true}
+		);
```

Comment:
This was an odd change I needed to make. Playwright was resolving 
```js
.getByRole('button', {name: 'Select Data Set'})` 
```
to the `Data Set` button in the left menu of fragment configuration.

Reply:
Also, I considered splitting locators and configuraiton logic into a new `DataSetFragmentConfigurationPage`, but the PR is already too big.

Reply:
I'd say this is fine, we own the selectors and the labels